console/src/main/java/org/georchestra/console/ws/utils/Validation.java

```diff
@@ -191,4 +196,62 @@ public boolean validateUrlFieldWithSpecificMsg(String fullyQualifiedField, Strin
         }
         return true;
     }
+
+    public boolean validateOrgUnicityByUniqueId(OrgsDao orgDao, JSONObject changes) {
+        String orgUniqueId = "";
+        String uuid = "";
+
+        if (changes == null) {
+            return true;
+        }
+        // case - not value to verify
+        if (!changes.has("orgUniqueId")) {
+            return true;
+        }
+
+        orgUniqueId = changes.getString("orgUniqueId");
+
+        if (orgUniqueId == null || orgUniqueId.isEmpty()) {
+            return true;
+        }
+
+        // test if org with same value already exists
+        Org findByOrgUniqueId = orgDao.findByOrgUniqueId(orgUniqueId);
+        // No org exists with this field value
+        if (findByOrgUniqueId == null) {
+            return true;
+        }
+
+        // No uuid to compare means that's an org creation
+        // and at this step, an org already exists with this orgUniqueId
+        if (!changes.has("uuid")) {
+            // we can't validate this orgUniqueId that already exists
+            return false;
+        }
+
+        uuid = changes.getString("uuid");
+
+        if (uuid.isEmpty() || uuid == null) {
+            return false;
+        }
+        // Control if this is same org update
+        return Objects.equals(findByOrgUniqueId.getUniqueIdentifier(), UUID.fromString(uuid));
+    }
+
+    public boolean validateOrgUniqueIdField(OrgsDao orgDao, JSONObject changes, Errors errors) {
+        Boolean isValid = true;
+        if (changes.has("orgUniqueId") && !this.validateOrgUnicityByUniqueId(orgDao, changes)) {
+            errors.rejectValue("orgUniqueId", "error.orgUniqueIdExists", "orgUniqueIdExists");
+            isValid = false;
```

Comment:
return false directly, with a retrun true below avoid variable instantiation but it's minor and can be kept like that

Reply:
or 
```suggestion
            boolean isNotValid = changes.has("orgUniqueId") && !this.validateOrgUnicityByUniqueId(orgDao, changes);
            if(isNotValid) errors.rejectValue("orgUniqueId", "error.orgUniqueIdExists", "orgUniqueIdExists");
            return isNotValid
```
also Boolean with an uppercase is not a primitive type and can accept null values. `boolean` is the primitive one and is false by default